.changeset/red-ants-lick.md

```diff
@@ -0,0 +1,39 @@
+---
+"xstate": minor
+"@xstate/react": minor
+---
+
+Added the ability to tighten TS declarations of machine with generated metadata. This opens several exciting doors to being able to use typegen seamlessly with XState to provide an amazing typing experience.
+
+The basics work like this:
```

Comment:
This is IMHO too technical and also describes a lot of implementation details that people shouldn't focus on, nor they should try to declare `Meta` type on their own.

Reply:
How would you word it?

Reply:
I would only mention how you can use this, that it's a beta feature, give a short example - but I would omit the `interface Meta` declaration from here